aries-core/src/js/components/external/stories/Accordion.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Accordion, AccordionPanel, Box, Grommet } from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'Accordion',
+};
+
+export const Simple = () => {
+  const pad = { vertical: 'medium' };
+  return (
+    <Grommet theme={aries} full>
+      <Accordion>
+        <AccordionPanel label="Our Company">
+          <Box pad={pad}>We are HPE.</Box>
+        </AccordionPanel>
```

Comment:
this could be simplified with an additional component AccordionPanelWrapper, and replace all similar duplications in this code.

Reply:
Good clean up, I'll do that.